src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -39,6 +39,17 @@ public CacheOptions(bool useSharedCache)
             UseSharedCache = useSharedCache;
         }
 
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="useSharedCache">Set to true to share the cache between all ClientApplication objects. The cache becomes static. <see cref="UseSharedCache"/> for a detailed description. </param>
+        /// <param name="sizeLimit">Token cache size limit in bytes. <see cref="SizeLimit"/> for a detailed description.</param>
+        public CacheOptions(bool useSharedCache, long sizeLimit)
```

Comment:
Can we have default values here (as per any recommended practices)?

Reply:
I don't think we can provide a realistic default value; it would be based on the user's scenario.

Reply:
Yeah, that's a good question we're struggling with :). What would be a good default? ...

Reply:
If default varies widely based on the user scenarios, we cannot do much. But if it primarly leans towards a certain value, that can be used. But we do not have answer, then it is good as it is.

Reply:
Danny also touched on a point of should there bit a minimum. Currently if someone specifies small value (like 1), cache will be compacted on each operation.